polaris-react/src/components/IndexTable/components/Row/Row.tsx

```diff
@@ -39,6 +39,8 @@ export interface RowProps {
    * @default "Select {resourceName}"
    */
   accessibilityLabel?: string;
+  /** Whether the row should have border-top */
+  borderless?: boolean;
```

Comment:
```suggestion
  border?: boolean;
```

Can we simplify the prop name to `border` with the default being `true`? When no border is desired, the usage would be `border={false}`. Using `border` is more aligned with the CSS semantic.

Sorry, I know that will also affect the class names and stories 😅

Reply:
After thinking more about it, using a string would be more flexible for future needs. Could we use `border="none"`?

Reply:
Thank you for your feedback, @sam-b-rose ! 
In case with `borderless` we add `TableRow-borderless` class and remove borders with css.
If you suggest  `border?: boolean;` or `border="none"`, how this should be styled? Should we by default add `TableRow-border` class?
With `border="none"` what other values it should have, `border="yes"`?
Thank you!